src/solvers/refinement/string_refinement_invariant.h

```diff
@@ -0,0 +1,15 @@
+/*******************************************************************\
+
+Module: Placeholder for eventual upgraded invariants.
+
+Author: Jesse Sigal, jesse.sigal@diffblue.com
+
+\*******************************************************************/
+
+#ifndef CPROVER_SOLVERS_REFINEMENT_STRING_REFINEMENT_INVARIANT_H
+#define CPROVER_SOLVERS_REFINEMENT_STRING_REFINEMENT_INVARIANT_H
+
+#define string_refinement_invariantt(reason) \
```

Comment:
It doesn't but @smowton's thing isn't done yet, and when it is I'll use that, so this is just temporary. The thing being diffblue/cbmc/pull/1063.